frontend/src/metabase/dashboard/actions/core.ts

```diff
@@ -85,8 +88,10 @@ export const onUpdateDashCardVisualizationSettings = createAction(
     id: DashCardId,
     settings: DashCardVisualizationSettings | null | undefined,
   ) => ({
-    id,
-    settings,
+    payload: {
```

Comment:
what's the consequence of changing the return value here? I couldn't figure out if we need to change any call sites. Can you explain why we need to do that here as well other similar change as well.

Reply:
ah right. This is a consequence of changing the `createAction` call from `redux-actions` to `@reduxjs/toolkit`. `redux-actions` adds the `payload` wrapper as a QoL improvement, but it has typings which get really messed up when we use RTK's `createReducer` and its typing system. 

So instead, I used the latter `createAction` which is typed more accurately to work with `createReducer`, but we lose the `payload` wrapper, so I had to add it back

Reply:
It doesn't affect the callsites (which I made sure to avoid so I wouldn't blow this PR out of proportion!) - everything should remain the same for these calls

Reply:
that's clear thanks!